communication/calendar-guidelines.md

```diff
@@ -95,6 +100,54 @@ will prevent you from getting updates like a canceled meeting. Join the main
 
 Accept the invite from the sender and you'll have the updates.
 
+### Calendar event template
+
+In case you are creating a calendar event and wondering what to include in it, you can refer to the following template
+
+#### Title
+
+```
+[SIG-<name>] <meeting title>
+or
+[WG-<name>] <meeting title>
+or
+[User Group-<name>]
+```
+#### Description
+
+```
+<One line intro about the SIG/ WG. Can include github links>
+
+Time: <Weekly/ biweekly/... or is the meeting for a fixed time and till when?>
+
+Meeting Notes & Agenda: <Hyperlink to the meeting notes doc>
+
+Zoom link: <Zoom invite link. Preferably both app and web client links>
+
+Additional Info: <Any extra info/ links can be provided here>
+
+Livestream link: <optional>
+
+Youtube Playlist: <optional>
+
+Google group: <optional>
+```
+
+#### Example
+
+> SIG contribex sub project, Contributor Marketing team weekly discussion
+> 
+> https://github.com/kubernetes/community/tree/master/communication/marketing-team
+> 
+> **Time**: Weekly. Every Friday
+> 
+> **Meeting Notes & Agenda**: https://docs.google.com/document/d/1IlHAJ131akGhI5ffF4OoVW0PrsVY4C0BB8l-UyQaQVo/edit
+> 
+> **Zoom link**:
+>
+> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; App: https://zoom.us/j/596959769
```

Comment:
```suggestion
> https://zoom.us/j/596959769
```

Reply:
I'd actually use `https://zoom.us/j/XXXXXXXXXX` just to ensure no one actually uses that meeting ID^^;;;

Reply:
I have added `&nbsp` to have the links indented and clear. I am assuming, a user will end up copy pasting it from rendered markdown. Or should we let it be without any indentation at all?

Reply:
Great point, @rajula96reddy -- I missed that when looking at plaintext.